experimental/cmd/sn3218/main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"log"
+	"time"
+
+	"periph.io/x/periph/conn/i2c/i2creg"
+	"periph.io/x/periph/experimental/devices/sn3218"
+	"periph.io/x/periph/host"
+)
+
+func main() {
+	if _, err := host.Init(); err != nil {
+		log.Fatal(err)
+	}
+
+	b, err := i2creg.Open("")
+
+	defer b.Close()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	l, _ := sn3218.New(b)
+	l.Reset()
+	l.Enable()
+
+	log.Println("Switch each LED on one by one")
```

Comment:
This is more of a smoke test. The executable should provide a simple way to control the LEDs.

Reply:
I simplified the main.go 
It's now just showing one call of each public function with a short comment. As it's a write only device it's a bit unclear how to build something that "has purpose" (maybe just blink 42 times ;-))